src/auditor_apel_plugin/core.py

```diff
@@ -315,16 +311,13 @@ def create_summary_db(config, records):
         if site_id not in sites_to_report:
             continue
 
-        if site_name_mapping is not None:
-            try:
-                site_name = site_name_mapping[site_id]
-            except KeyError:
-                logging.critical(
-                    f"No site name mapping defined for site {site_id}"
-                )
-                raise KeyError
-        else:
-            site_name = site_id
+        try:
+            site_name = site_name_mapping[site_id]
+        except KeyError:
+            logging.critical(
+                f"No site name mapping defined for site {site_id}"
+            )
+            raise KeyError
```

Comment:
Is it intended that you raise a bare `KeyError` without a message here?

Maybe state why the exception is raised, i.e.
```
raise KeyError("<your error message here>")
```

Reply:
fixed as discussed